### PR TITLE
Adding a small workaround in logging: --quiet option

### PR DESCRIPTION
**Pull request by akuleshov7**

### CODE REVIEW

File: save-core/src/commonNonJsTest/kotlin/org/cqfn/save/core/integration/ClassicWarnTest.kt [hunk 1]
```diff
@@ -41,6 +41,17 @@ class ClassicWarnTest {
                 ), 3
             )
 
+    @Test
+    @Ignore
```
Comment (akuleshov7) [inline save-core/src/commonNonJsTest/kotlin/org/cqfn/save/core/integration/ClassicWarnTest.kt:45]: this test will be fixed in 277

File: buildSrc/src/main/resources/config-options.json [hunk 2]
```diff
@@ -28,7 +28,7 @@
       "kotlinType": "kotlin.Boolean",
       "fullName" : "quiet",
       "shortName" : "q",
-      "description" : "Do not log anything",
+      "description" : "Log only test results without any warnings",
```
Comment (petertrr) [inline buildSrc/src/main/resources/config-options.json:31]:
```suggestion
      "description" : "Log only test results (if `--result-output` is `stdout`) without any warnings",
```

File: OptionsTable.md [hunk 3]
```diff
@@ -6,7 +6,7 @@ Most (except for `-h` and `-prop`) of the options below can be passed to a SAVE
 | parallel | parallel-mode | Whether to enable parallel mode | false |
 | t | threads | Number of threads | 1 |
 | d | debug | Turn on debug logging | false |
-| q | quiet | Do not log anything | false |
+| q | quiet | Log only test results without any warnings | false |
```
Comment (petertrr) [inline OptionsTable.md:9]: Will errors still be logged? It's not clear from the description

File: OptionsTable.md [hunk 4]
```diff
@@ -6,7 +6,7 @@ Most (except for `-h` and `-prop`) of the options below can be passed to a SAVE
 | parallel | parallel-mode | Whether to enable parallel mode | false |
 | t | threads | Number of threads | 1 |
 | d | debug | Turn on debug logging | false |
-| q | quiet | Do not log anything | false |
+| q | quiet | Log only test results (if `--result-output` is `stdout`) without any warnings (only with critical errors) | false |
```
Comment (petertrr) [inline OptionsTable.md:9]: I think it's different now, from what's in json

**Review by petertrr — APPROVED** [review]

File: OptionsTable.md (hunk 4 above)
Comment (akuleshov7) [reply OptionsTable.md:9]: updated, thanks

**Pull request merged**